cmd/cmd.go

```diff
@@ -74,8 +82,25 @@ func appHandler(ctx *cli.AppContext) error {
 	if fileProvided && !fileFound {
 		return fmt.Errorf("file path was not found or inaccessible")
 	}
+
+	overwrite, err := flags.Bool("overwrite")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I think all log entries are not necessary in this case since we are returning every error anyways.